README.md

```
@@ -31,12 +31,24 @@
 </p>
 
-> Open-source AI Agent that empowers data-driven teams to chat with their data to generate Text-to-SQL, charts, spreadsheets, reports, and BI. 
+> Open-source GenBI AI Agent that empowers data-driven teams to chat with their data to generate Text-to-SQL, charts, spreadsheets, reports, and BI. 
 
 <p align="center">
   <img src="./misc/wren_workflow.png">
 </p>
 
+## 🕶 Try it yourself!
+
+#### GenBI (Generative Business Intelligence)
```

Markdownlint (0.37.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)